README.md

```diff
@@ -14,9 +14,9 @@ Supported receivers includes:
 ## CustomResourceDefinitions
 Notification Manager uses the following CRDs to define the desired alerts/notifications webhook and receiver configs.
 - NotificationManager: Defines the desired alerts/notification webhook deployment. The Notification Manager Operator ensures a deployment meeting the resource requirements is running.
-- EmailConfig: Defines the email configs like SmartHost, AuthUserName, AuthPassword, From, RequireTLS etc. 
+- EmailConfig: Defines the email configs like SmartHost, AuthUserName, AuthPassword, From, RequireTLS etc. There are also global options like NotificationTimeout, DeliveryType, MaxEmailReceivers to define email configs.
```

Comment:
In quick start section, there are options yaml snippet like below, is this format still valid after you change the option structure?

```
    options:
      notificationTimeout:
        email: 5
```